HISTORY.rst

```diff
@@ -5,6 +5,14 @@ History
 Unreleased
 ----------
 
+1.9.4 (2019-16-09)
```

Comment:
I was just talking to Alessio and we need to do a release with this bugfix ASAP, because it is generating different docs everytime the gecko view team creates a new PR. In this case is it ok to leave it as is?

Reply:
Sure, no problem.  One small nit, though -- I think the date should be `2019-10-16`.